quipucords/tests/scanner/network/test_network_inspect.py

```diff
@@ -86,7 +85,7 @@ def setup_method(self, _test_method):
 
         self.scan_task.update_stats("TEST NETWORK INSPECT.", sys_failed=0)
 
-        self.fact_endpoint = "http://testserver" + reverse("v1:reports-list")
+        self.fact_endpoint = "http://testserver/whatever"
```

Comment:
What even is the point of `self.fact_endpoint`? 🤔  I see it's used once for a mock call in `test_inspect_scan_fail_no_facts`, but I don't understand what actual purpose it serves there. I commented out the line and ran the test locally, and it still passed without it…

I know question is growing beyond the scope of your diff, but it might mean deleting a couple more lines, and we like that, right? 🙂

Reply:
I have the same question.
No actual endpoint is being called there from what I can tell. I changed to whatever to highlight how useless at least one mock is in that test.

If I wasn't too lazy to try to understand that test, I'd have a better answer. My gut feeling was to actually axe it, but I can't do that without understanding it first 😅